sydent/util/emailutils.py

```diff
@@ -83,14 +83,25 @@ def sendEmail(
             mailString = template_file.read() % allSubstitutions
 
     parsedFrom = email.utils.parseaddr(mailFrom)[1]
+    if parsedFrom == "":
+        # This address is part of the application config, so errors here are
+        # internal server errors.
+        raise RuntimeError(
```

Comment:
Should we parse the `mailFrom` during start and raise config errors instead?

Reply:
We should. I was being lazy, but let's do it properly.

Reply:
I've pulled out the parsing/validation into a free function, and called that both here and from the config parsing logic. What do you think of it?